Create models. Make migrations.

lewisemm opened the pull request:
- Created initial models for User, Member, Project, Team, Story and Task
- Removed 'db.sqlite3' file and replaced database configurations at 'limber/settings.py'
- Created initial migrations for models created

[Reference #107204654]

lewisemm committed:
Create models. Make migrations.

- Created initial models for User, Member, Project, Team, Story and Task
- Removed 'db.sqlite3' file and replaced database configurations at 'limber/settings.py'
- Created initial migrations for models created

[Reference #107204654]

andela-anandaa commented:
Fix the failing tests first.

@andela-engmkwalusimbi - you will have to pull the branch directly to yours, I won't merge a failing branch.

Commit:
Get DATABASE_URL variable from environment

- Create .env file to store environment variables on virtualenv

[Reference #107204654]

Commit:
Create new migrations from new models.

- Add '.env' file to gitignore.

[Reference #107204654]
